deployments/get/objects.go

```diff
@@ -352,6 +352,38 @@ func GetHostPolicyManagerDeployment(namespace string) *appsv1.Deployment {
 	}
 }
 
+var gkeHostUsrVolMnt = corev1.VolumeMount{
+	Name:      "usr-src-path", // /usr -> /media/root/usr (read-only) check issue #579 for details
+	MountPath: "/media/root/usr",
+	ReadOnly:  true,
+}
+
+var gkeHostUsrVol = corev1.Volume{ // check #579 why GKE is handled separately
```

Comment:
We should keep environment specific mounts in defaultConfigs Map. https://github.com/kubearmor/KubeArmor/blob/925a26abe2d61d17f364d5f942184a4a0b7671c3/deployments/get/defaults.go#L296

Reply:
Interesting.

So in this case, I have to copy VolumeMounts and Volumes that are "gke" specific in "gke" and also copy the default ones in every other case (k3s, eks, minikube....) ... is this understanding correct?

Reply:
Handled.